docs/content/usage.md

```diff
@@ -27,46 +87,34 @@ All workflows are using the tagging convention listed below. It is subsequently
 wget "${DMS_GITHUB_URL}/docker-compose.yml"
 wget "${DMS_GITHUB_URL}/mailserver.env"
-
-# Optional
-wget "${DMS_GITHUB_URL}/setup.sh"
-chmod a+x ./setup.sh
 ```
 
-## Create a docker-compose Environment
+Then edit `docker-compose.yml` to your liking; substitute `mail.example.com` according to your FQDN, and if you want to use SELinux for the `./docker-data/dms/config/:/tmp/docker-mailserver/` mount, append `-z` or `-Z`. Then configure the environment specific to the mail server by editing [`mailserver.env`][docs-environment], but keep in mind that:
 
-1. [Install the latest Docker Compose](https://docs.docker.com/compose/install/)
-2. Edit `docker-compose.yml` to your liking
-    - substitute `mail.example.com` according to your FQDN
-    - if you want to use SELinux for the `./docker-data/dms/config/:/tmp/docker-mailserver/` mount, append `-z` or `-Z`
-3. Configure the mailserver container to your liking by editing `mailserver.env` ([**Documentation**](../config/environment/)), but keep in mind this `.env` file:
-    - [_only_ basic `VAR=VAL`](https://docs.docker.com/compose/env-file/) is supported (**do not** quote your values)
-    - variable substitution is **not** supported (e.g. :no_entry_sign: `OVERRIDE_HOSTNAME=$HOSTNAME.$DOMAINNAME` :no_entry_sign:)
+- [_only_ basic `VAR=VAL`][docker-compose-env-file] is supported (**do not** quote your values)
+- variable substitution is **not** supported (e.g. :no_entry_sign: `OVERRIDE_HOSTNAME=$HOSTNAME.$DOMAINNAME` :no_entry_sign:)
 
-!!! info "Podman Support"
-
-    If you're using podman, make sure to read the related [documentation](../config/advanced/podman/)
-
-## Get up and running
+[docs-environment]: ./config/environment.md
+[docker-compose-env-file]: https://docs.docker.com/compose/env-file/
 
-### First Things First
+### Get Up and Running
 
 !!! danger "Using the Correct Commands For Stopping and Starting DMS"
 
     **Use `docker compose up / down`, not `docker compose start / stop`**. Otherwise, the container is not properly destroyed and you may experience problems during startup because of inconsistent state.
 
     Using `Ctrl+C` **is not supported either**!
 
-You are able to get a full overview of how the configuration works by either running:
-
-1. `./setup.sh help` which includes the options of `setup.sh`.
-2. `docker run --rm docker.io/mailserver/docker-mailserver:latest setup help` which provides you with all the information on configuration provided "inside" the container itself.
+You are able to get a full overview of how the configuration works by running `docker run --rm ghcr.io/docker-mailserver/docker-mailserver:${TAG} setup help`, which provides you with all the information on configuration.
 
 ??? info "Usage of `setup.sh` when no DMS Container Is Running"
 
+    We encourage you to directly use `setup` inside the container (for instructions, see `docker exec -ti <CONTAINER NAME> setup help`). If you still want to use `setup.sh`, here's some information about it.
```

Comment:
> This is just repeating the same command as above almost, but with `docker exec` instead of `docker run`.

Indeed, but I want to make sure users understand that they actually do not really need `setup.sh` anymore :)

> I imagine most users would just use `docker exec` with `bash` to run multiple `setup ...` commands, or with `setup` instead of `bash` if automating through some scripts.

Probably 🤔

> `setup.sh` is mostly complimentary at this point, and not likely to be too useful for most users? (_beyond saving a few key strokes which an alias could do just as well_)

Yes, I think so too.

Reply:
```suggestion
    We encourage you to directly use `setup` inside the container (like shown above). If you still want to use `setup.sh`, here's some information about it.
```